homeassistant/components/sensor/point.py

```diff
@@ -0,0 +1,146 @@
+"""
+Support for Minut Point.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.point/
+"""
+import logging
+
+from homeassistant.core import callback
+from homeassistant.const import (
+    DEVICE_CLASS_HUMIDITY, DEVICE_CLASS_ILLUMINANCE,
+    DEVICE_CLASS_TEMPERATURE, DEVICE_CLASS_PRESSURE,
+    TEMP_CELSIUS)
+from homeassistant.helpers.entity import Entity
+from homeassistant.helpers.dispatcher import async_dispatcher_connect
+from homeassistant.util.dt import utcnow, parse_datetime
+
+DOMAIN = 'point'
+
+_LOGGER = logging.getLogger(__name__)
+
+DEVICE_CLASS_SOUND = 'sound_level'
+DEVICE_CLASS_AMBIENT_LIGHT = 'part_als'
+
+SENSOR_TYPES = [DEVICE_CLASS_TEMPERATURE, DEVICE_CLASS_PRESSURE,
+                DEVICE_CLASS_HUMIDITY, DEVICE_CLASS_SOUND]
+
+
+async def async_setup_entry(hass, config_entry, async_add_entities):
+    """Setup a Point config entry."""
+    _LOGGER.debug("Async setup: %s", config_entry.data)
+
+    device_id = hass.data['point_device']
+    client = hass.data[DOMAIN][config_entry.entry_id]
+    async_add_entities((MinutPointSensor(hass, client, device_id, sensor_type)
+                        for sensor_type in SENSOR_TYPES), True)
+
+
+class MinutPointSensor(Entity):
+    """The platform class required by Home Assistant."""
+
+    def __init__(self, hass, point_client, device_id, sensor_type):
```

Comment:
Don't pass in `hass`. It will be set on the entity when it has been added to home assistant.